content/kubermatic/main/installation/install-kkp-ee/_index.en.md

```diff
@@ -57,6 +57,17 @@ spec:
     }
 ```
 
+```yaml
+kubermaticOperator:
+  # insert the Docker authentication JSON provided by Kubermatic here
+  imagePullSecret: |
+    {
+      "auths": {
+        "quay.io": {....}
+      }
+    }
+```
```

Comment:
I don't think this is necessary, because the installer will fill this in from the passed `KubermaticConfiguration`. At least all our EE setups don't need this in their `values.yaml`.

Reply:
At the moment, we have it [here](https://github.com/kubermatic/kubermatic/blob/main/charts/values.example.ee.yaml#L75) included.